package.json

```diff
@@ -1,16 +1,44 @@
-{
-  "name": "task1",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.3.1",
-    "react-dom": "^16.3.1",
-    "react-scripts": "1.1.4"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
-  }
-}
\ No newline at end of file
+{
+    "name": "react_task2",
+    "version": "1.0.0",
+    "description": "App built as task 2 for the react mentoring program",
+    "main": "index.js",
+    "scripts": {
+        "build": "webpack --config ./webpack.config.js --mode production --env.production",

```

Comment:
You can create some smaller npm scripts and reuse theirs to do not repeat same commands with different flags all the time

Reply:
Done.